src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js

```diff
@@ -84,14 +84,16 @@ const makeId = makeGenerator()
  * "longitude": -122.45,
  * "zoom": 15,
  * "controller": true,
+ * "isPickable": true,
  * "layers": [{
  *     "type": "Scatterplot_Layer",
  *     "data": [{
  *         "type": "Geo_Point",
  *         "latitude": -122.45,
  *         "longitude": 37.8,
  *         "color": [255, 0, 0],
- *         "radius": 100
+ *         "radius": 100,
+ *         "message": "foo bar baz"
```

Comment:
`message` is not very descriptive. It might be better to call it `label`, `text_label`or `tooltip`. Maybe have a look what the name in other visualization system is and use something equivalent.

Reply:
+1 for label